Cargo.toml

```diff
@@ -6,7 +6,7 @@ repository = "https://github.com/tari-project/randomx-rs"
 homepage = "https://tari.com"
 readme = "README.md"
 license = "BSD-3-Clause"
-version = "0.4.1"
+version = "0.4.2"
```

Comment:
Think this should be 0.5 since it a breaking change for existing errors.

Reply:
The public interface didn't change though